Разметка главной страницы

EvgeniyaDen opened the pull request.

EvgeniyaDen committed:
Разметка главной страницы

1kazakov commented on index.html:
```diff
@@ -2,11 +2,157 @@
+          <li class="navigation-buttons-item">
+            <a class="navigation-buttons-link" href="#">
+                Хочу сюда!
+            </a>
```
Почему здесь ссылка ?

1kazakov replied:
все правильно

1kazakov commented on index.html:
```diff
@@ -2,11 +2,157 @@
+      <section class="hero">
+        <h1 class="visually-hidden">Отдых в городке Седона</h1>
+        <img class="hero-background-img" src="#" alt="Городок Седона">
+        <ul class="title-list">
```
какие критерии объединения элементов в список?

1kazakov commented on index.html:
```diff
@@ -2,11 +2,157 @@
+      <section class="search">
+        <h2 class="search-title">Заинтересовались?</h2>
+        <p class="search-description">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="search-button" href="catalog.html">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</a>
```
по хорошему текст должен быть как мы привыкли его читать, первая заглавная остальные строчные

1kazakov commented on index.html:
```diff
@@ -2,11 +2,157 @@
+        </ul>
+      </section>
+      <section class="search">
+        <h2 class="search-title">Заинтересовались?</h2>
```
действительно ли это заголовок?

1kazakov commented on index.html:
```diff
@@ -2,11 +2,157 @@
+      </section>
+    </main>
+    <footer class="page-footer">
+      <section class="footer-form">
```
почему это секция и почему в футере?

1kazakov commented on index.html:
```diff
@@ -2,11 +2,157 @@
+          </li>
+        </ul>
+        <a class="contacts-phone" href="tel:88128121212">+7 (812) 812-12-12</a>
+        <a class="footer-logo" href="#">
```
лучше добавить скрытый спан с подписью что это разработчик

EvgeniyaDen committed:
Добавлено количество в "избранное"

EvgeniyaDen committed:
Удален список для 3-х векторных изображений

EvgeniyaDen committed:
Добавлен скрытый заголовок в раздел "поиск гостиницы"

EvgeniyaDen committed:
Удалены img для соцсетей

EvgeniyaDen committed:
Добавлен скрытый span в лого в футере

EvgeniyaDen committed:
Добавлен <label> в форму

EvgeniyaDen committed:
Разметка заголовка и ХК в каталоге

EvgeniyaDen committed:
Верстка секции с фильтрами

EvgeniyaDen committed:
разметка карточек гостиниц и пагинации

EvgeniyaDen committed:
Форма перенесена из футера в мэйн

1kazakov commented on catalog.html:
```diff
@@ -0,0 +1,280 @@
+            <input class="filter-range" type="range" id="price-range" min="0" max="9000" value="7000">
+          </fieldset>
+          <button class="apply-button" type="submit">Применить</button>
+          <button class="reset-button" type="reset">Сбросить</button>
```
отлично что тут сразу тип reset сделала 👍

1kazakov commented on catalog.html:
```diff
@@ -0,0 +1,280 @@
+            <legend class="hotel-filter-title">Стоимость, ₽</legend>
+            <label class="visually-hidden" for="price-from">Стоимость, от</label>
+            <input type="number" name="price-from" id="price-from" min="0" placeholder="0" value="0">
+            <label class="visually-hidden" for="price-to">Стоимость, до</label>
```
тут немного каши

1kazakov commented on catalog.html:
```diff
@@ -0,0 +1,280 @@
+        <select class="select-control" aria-label="Сортировка">
+          <option value="cheap">Сначала дешевые</option>
+        </select>
+        <!--в примере делают через <a>, сделала как в примере, но есть сомнения, что это всё же button-->
```
забыл тебе еще в прошлый раз напомнить, что все в название чего мы можем добавить слово список скорее всего является списком. Список кнопок отображения карточек 👍
на счет ссылок и кнопок на созвоне обсудим

1kazakov commented on catalog.html:
```diff
@@ -0,0 +1,280 @@
+            <a class="hotels-card-link" href="#">
+              <img class="hotels-card-image" src="#" alt="">
+            </a>
+            <a class="hotels-card-link">
```
у тебя тут ссылка без хреф, и не лучше ли сделать одну ссылку вокруг картинки и названия?

EvgeniyaDen committed:
Исправлены недочеты

Добавлены ссылки-заглушки, переключатель отображения карточек обернут в список, исправлены лейблы цены от и до. И  не заметила мелочь, одна кнопка должна быть "в избранном" вместо "в избранное" (т.е. отель уже добавлен в избранное), тоже поправила

EvgeniyaDen committed:
Графика оптимизирована, загружена в папки

EvgeniyaDen committed:
Подключены контентные изображения в разметке
